Sources/SwiftGenKit/Parsers/AnyCodable.swift

```diff
@@ -0,0 +1,129 @@
+//
+// SwiftGen
+// Copyright © 2020 SwiftGen
+// MIT Licence
+//
+
```

Comment:
```suggestion

// Original credits to Ian Keen (https://gist.github.com/IanKeen/df3bed29e2613ead94aa3b16967b9d14)

```

Reply:
@IanKeen Is it ok with you if we credit your gist this way while including it in our MIT-licensed code?

Reply:
Of course! 🤘

Reply:
@IanKeen feel free to double-check my changes to your gist [here](https://github.com/SwiftGen/SwiftGen/pull/779/commits/bd3644d5bdd4c934de1404bbacbed9d44e6a7bba) – a second pair of eyes is always welcome, especially from the Codable expert in the room 😛 

(Another diff from our code and your gist is that I've added `Date` as a type supported by `AnyCodable` – since Plist are able to hold date instances after all)